src/software/ai/evaluation/detect_successful_kick.cpp

```diff
@@ -0,0 +1,16 @@
+
+#include "detect_successful_kick.h"
+
+bool successfulKickDetected(const Ball& ball, const Vector& expected_kick_direction)
+{
+    double min_kick_speed       = 0.5;
+    double max_angle_difference = 20;
```

Comment:
this should be static constexpr and ALL_CAPS

Reply:
done for `MAX_ANGLE_DIFFERENCE`, `min_kick_speed` moved to parameters